purchase.adoc

```diff
@@ -375,3 +404,97 @@ purchaseUUID:: The UUID of the purchase
     "refunded": false
 }
 ```
+
+### Discounts
+Discounts can exist both on a specific row of items in the shopping cart, as well as on the entire shopping cart. A discount can be set either as a percentage or as a fixed value.
+
+When applying the total discounts on the purchase price, the discounts set on rows will be calculated first and subtracted from the total price followed by any discount set on the entire chart.
```

Comment:
While correct, I wonder if it is not easier to express as something you calculate per row. 

The difference is that you calculate the discounted price per row and sum those together, rather than calculate the sum of undiscounted row prices and the the sum of the row discounts and then subtract those.

I think it makes for a more understandable mental model, at least for me. :)